test/jdk/tools/jlink/runtimeImage/BasicJlinkTest.java

```diff
@@ -39,7 +39,7 @@
  *          jdk.jlink/jdk.tools.jimage
  * @build tests.* jdk.test.lib.process.OutputAnalyzer
  *        jdk.test.lib.process.ProcessTools
- * @run main/othervm -Xmx1400m BasicJlinkTest false
+ * @run main/othervm -Xmx1400m BasicJlinkTest
```

Comment:
Note to reviewers: This is just a clean-up of a parameter that isn't being used, so I've removed it here.